### PR TITLE
Homework 3

### PR DESCRIPTION
**Pull request by ghost-lv**

### CODE REVIEW

**Comment by Volodymyr95** [comment]
var rnd = r.nextInt(3);
+        var Symbol = "";
+        var Result = 0;
+        var ResultString =""; variable names should starts from lower case letter

<sub>Sent with <a href="http://githawk.com">GitHawk</a></sub>

**Comment by Volodymyr95** [comment]
+        var rnd = r.nextInt(3);
+        var Symbol = "";
+        var Result = 0;
+        var ResultString ="";
+        if (rnd==0) {
+            Symbol = "-";
+        }
+        else if (rnd==1) {
+            Symbol="+";
+        }
+        else if (rnd==2){
+            Symbol="*";
+        }
+        var a =0;
+        var b =0;
+        if (Symbol == "-") {
+            a=5;
+            b=6;
+        }
+        else if (Symbol=="+"){
+            a=15;
+            b=22;
+        }
+        else if (Symbol=="*") {
+            a = 30;
+            b = 20;
 Not correct 

<sub>Sent with <a href="http://githawk.com">GitHawk</a></sub>

File: src/com/lits/hw3/hw3Main.java [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+
+import java.util.Random;
+
+public class hw3Main {
+    public static void main(String[] args) {
+        //У вас є формула : a+11/12(b+b)/12 і є символ,
+        // якшо символ ‘-’ ви ініціалізуєете змінні одними парамеитрами,
+        // якшо ‘+’ то ще іншими і якшо ‘*’ іншими
+        Random r = new Random();
+        var rnd = r.nextInt(3);
```
Comment (Volodya227) [inline src/com/lits/hw3/hw3Main.java]: в програмі пролітає число 3, якщо випаде

**Comment by ghost-lv** [comment]
updated

**Comment by ghost-lv** [comment]
Наталя, ти стерла бренч )

**Comment by ghost-lv** [comment]
updated simple numbers

**Review by natalia-kozak — APPROVED** [review]

File: src/com/lits/hw3/hw3Main.java [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+    public static void main(String[] args) {
+        //У вас є формула : a+11/12(b+b)/12 і є символ,
+        // якшо символ ‘-’ ви ініціалізуєете змінні одними парамеитрами,
+        // якшо ‘+’ то ще іншими і якшо ‘*’ іншими
+        Random r = new Random();
+        var rnd = r.nextInt(2);
+        var symbol = (char)(0);
+        var result = 0;
+        var resultString ="";
+        if (rnd==0) {
```
Comment (Volodymyr95) [inline src/com/lits/hw3/hw3Main.java:15]: u should check symbols, not numbers

**Review by Volodymyr95 — COMMENTED** [review]
need to fix 1st task

File: src/com/lits/hw3/hw3Main.java (hunk 2 above)
Comment (ghost-lv) [reply src/com/lits/hw3/hw3Main.java:15]: dont know how to generate rnd symol from enum ...yet

Comment (Volodymyr95) [reply src/com/lits/hw3/hw3Main.java:15]: we don't need to generate random numbers, just create char variable, and use it for equal with special symbols

**Pull request merged**